### PR TITLE
chore(spell): Refactor aqua

### PR DESCRIPTION
**Pull request by InversionSpaces**

### CODE REVIEW

**Review by justprosh — APPROVED** [review]
Cool!

File: src/aqua/installation-spell/src/aqua/cli.aqua [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
 
 import Srv from "@fluencelabs/aqua-lib/builtin.aqua"
 import "@fluencelabs/aqua-lib/subnet.aqua"
 import Spell, Log from "@fluencelabs/spell/spell_service.aqua"
 
+import "./config.aqua"
+import deploy from "./deploy.aqua"
+import "./console.aqua"
+import "./deal_spell.aqua"
+import upload_workers, upload_deal from "./upload.aqua"
```
Comment (folex) [inline src/aqua/installation-spell/src/aqua/cli.aqua:13]: Would be super nice to always use `import from`, and never use wildcard imports

**Review by folex — APPROVED** [review]
nice, thank you!

File: src/aqua/installation-spell/src/aqua/upload.aqua [hunk 2]
```diff
@@ -1,8 +1,13 @@
+aqua Upload declares *
```
Comment (kmd-fl) [inline src/aqua/installation-spell/src/aqua/upload.aqua:1]: A bit unrelated question: why do you choose the `aqua` keyword, which doesn't have any meaning, and not `lib`, which is exactly what this is? 
imho this looks more understandable the original
```aqua
lib Upload declares *
```

**Review by kmd-fl — APPROVED** [review]

**Pull request merged**